### ISSUE TITLE
WCAG 2.0 or 2.1?

### ISSUE THREAD

**Issue by murata2makoto** [mattgarrish/epub-revision#2099]
Are the two bullets in [3.3.1 in EPUB Accessibility 1.1](https://w3c.github.io/epub-specs/epub33/a11y/#sec-wcag-conf) talking about WCAG 2.0 or 2.1?

The first bullet is:

>MUST meet the requirements of WCAG 2.0 [[WCAG20](https://www.w3.org/TR/epub-a11y-11/#bib-wcag20)], but it is strongly RECOMMENDED that it meet the requirements of the [latest recommended version of WCAG 2](https://www.w3.org/TR/WCAG2/).

So conformance to 2.0 is a must, while that to 2.1 is not.

>MUST meet the requirements of [Level A](https://www.w3.org/TR/WCAG2/#cc1), but it is strongly RECOMMENDED that it meet the requirements of [Level AA](https://www.w3.org/TR/WCAG2/#cc1) [[WCAG2](https://www.w3.org/TR/epub-a11y-11/#bib-wcag2)].

Here the link for "Level A" references WCAG 2.1.  But it shouldn't, since "Leve A" in WCAG 2.0 and that in 2.1 are different.

**Comment by mattgarrish** [comment]
> MUST meet the requirements of WCAG 2.0 [[WCAG20](https://www.w3.org/TR/epub-a11y-11/#bib-wcag20)], but it is strongly RECOMMENDED that it meet the requirements of the [latest recommended version of WCAG 2](https://www.w3.org/TR/WCAG2/).

That should be a lowercase "recommended". It's guidance not normative.

> So conformance to 2.0 is a must, while that to 2.1 is not.

Yes, we're not requiring people to move to 2.1.

> Here the link for "Level A" references WCAG 2.1.

This is trickier because no matter what version you choose to conform to, you must meet Level A. If link to WCAG 2.0 Level A/AA, then it's equally confusing why those would be the requirements if you're conforming, for example, to 2.1.

Maybe we can improve this by clearly saying it's version dependent:

> Regardless of the version chosen, ...

And then replace the links with a note that says to refer to the respective version for the conformance level requirements. The levels are such central concepts it's not like this will be confusing to anyone if we don't link them.
